webapp/tests/karma/ts/modules/error/error.component.spec.ts

```diff
@@ -45,6 +49,11 @@ describe('ErrorComponent', () => {
           { provide: DOCUMENT, useValue: documentMock },
         ]
       })
+      .overrideComponent(ErrorComponent, {
+        set: {
+          template: `<div></div>`
```

Comment:
Well played! 😁 
Just a comment on the type of quote
```suggestion
          template: '<div></div>'
```